build/source/driver/multi_driver.f90

```diff
@@ -211,23 +212,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
+character(len=10)                :: strTMP                     ! local temp string
 ! *****************************************************************************
 ! (1) inital priming -- get command line arguments, identify files, etc.
 ! *****************************************************************************
-print*, 'start'
 ! get the initial time
-call date_and_time(cdate1,ctime1)
-print*,ctime1
+call date_and_time(values=ctime1)
+print "(A,I2,':',I2,':',I2)", 'start at ',ctime1(5:7)
+! check numbers of command-line arguments
+select case (iargc())
+ case (2)
+  !do nothing; this is a full run
+  strtHRU=0
+ case (4)
+  !get the starting hru and count
+  call getarg(3,strTMP); read(strTMP,*) strtHRU
+  call getarg(4,strTMP); read(strTMP,*) nHRU
+  if (strtHRU<1 .or. nHRU<1) then
+   err = 1
+   message = ' illegal startHRU or countHRU specification.'
+  else
+   print '(x,I0,A)', nHRU,' HRUs are selected for simulation.'
+  end if
+ case default
+  print*, 'Usage: summa.exe file_suffix master_file <startHRU countHRU>'
+  print*, '  summa.exe   -- summa excutable'
+  print*, '  file_suffix -- text string defining the output file suffix'
+  print*, '  master_file -- path/name of muster file'
```

Comment:
'Muster' was copied from the original warning print. Was it intentionally used because it has passed so many PR reviews.